Capstone Project/ViewControllers/Workouts/CurrentWorkoutViewController.m

```diff
@@ -148,18 +147,27 @@ - (nonnull UITableViewCell *)tableView:(nonnull UITableView *)tableView cellForR
     ExerciseCell *cell = [self.tableView dequeueReusableCellWithIdentifier:@"Exercise Cell" forIndexPath:indexPath];
     Exercise *exercise = self.arrayOfExercises[indexPath.row];
     cell.nameLabel.text = exercise.name;
-    cell.weightLabel.text = [@(exercise.weight) stringValue];
-    cell.set1Label.text = [@(exercise.set1) stringValue];
-    cell.set2Label.text = [@(exercise.set2) stringValue];
-    cell.set3Label.text = [@(exercise.set3) stringValue];
-    cell.set4Label.text = [@(exercise.set4) stringValue];
-    cell.set5Label.text = [@(exercise.set5) stringValue];
+    cell.weightLabel.text = [NSString stringWithFormat:@"%f", exercise.weight];
+    cell.set1Label.text = [NSString stringWithFormat:@"%d", exercise.set1];
+    cell.set2Label.text = [NSString stringWithFormat:@"%d", exercise.set2];
+    cell.set3Label.text = [NSString stringWithFormat:@"%d", exercise.set3];
+    cell.set4Label.text = [NSString stringWithFormat:@"%d", exercise.set4];
+    cell.set5Label.text = [NSString stringWithFormat:@"%d", exercise.set5];
```

Comment:
what happens when the number of sets is less than 5

Reply:
![image](https://user-images.githubusercontent.com/88735772/181415382-80358a2b-4316-44e8-9d7e-9054575bf105.png)